webcompat/form.py

```diff
@@ -202,20 +215,23 @@ def build_formdata(form_object):
         'url': form_object.get('url'),
         'browser': form_object.get('browser'),
         'os': form_object.get('os'),
-        'problem_type': get_problem(form_object.get('problem_category')),
-        'description': form_object.get('description')
+        'problem_type': get_radio_button_label(
+            form_object.get('problem_category'), problem_choices),
+        'browser_test_type': get_radio_button_label(form_object.get(
+            'browser_test'), tested_elsewhere),
+        'description': form_object.get('description'),
+        'steps_reproduce': form_object.get('steps_reproduce')
     }
 
     # Preparing the body
     body = u'''{metadata}
 **URL**: {url}
+**Problem type**: {problem_type}
+**Description**: {description}
+**Steps to Reproduce** {steps_reproduce}
```

Comment:
@miketaylr as long as it is helpful for the work on the bugs, I'd +1 for changing the flow in the report!